osutil/export_test.go

```diff
@@ -194,12 +195,9 @@ func MockEtcFstab(text string) (restore func()) {
 
 // MockUname mocks syscall.Uname as used by MachineName and KernelVersion
 func MockUname(f func(*syscall.Utsname) error) (restore func()) {
-	old := syscallUname
+	restore = testutil.BackupBeforeMocking(&syscallUname)
 	syscallUname = f
-
-	return func() {
-		syscallUname = old
-	}
+	return
```

Comment:
nitpick: I would find it more readable if this was an explicit `return restore`; see also https://go-review.googlesource.com/#/c/20024/

Reply:
Yesh, it's also the pattern we use in the code, even tough the return argument is named, the `return` explicitly includes the values that are actually returned.